style/style.css

```diff
@@ -1,3 +1,576 @@
-#test {
-  color: red;
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
+}
+
+.logBar {
+  display: none;
+}
+
+header {
+  position: fixed;
+  width: 100%;
+  z-index: 3;
+  box-shadow: 0 20px 20px rgba(0, 0, 0, 0.6);
+}
+
+nav {
+  display: none;
+}
+
+#topMenu {
+  display: flex;
+  align-items: center;
+  background-color: #fff;
+  padding: 0.5rem;
+}
+
+.logoHeader {
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+  padding-left: 0.5rem;
+}
+
+#logo {
+  height: 3.5rem;
+  width: 85%;
+}
+
+#menu {
+  width: 95%;
+  position: absolute;
+  list-style: none;
+  top: 5rem;
+  display: none;
+  background-image: linear-gradient(to bottom right, #414246, black);
+  background-color: #414246;
+  color: white;
+  box-shadow: 20px 20px 20px rgba(0, 0, 0, 0.9);
+  padding: 1rem;
+}
+
+#closeMenuBtn {
+  font-size: 2rem;
+  position: absolute;
+  right: 0;
+  margin-right: 1.5rem;
+}
+
+#menuButton {
+  height: 31px;
+  width: 27px;
+  padding: 0.2rem;
+}
+
+.menuItems {
+  text-align: center;
+  padding: 1rem;
+}
+
+.menuItems a {
+  color: #fff;
+  text-decoration: none;
+}
+
+#home {
+  background-image: url(../assets/mainMobileBack.svg);
+  background-size: 100%;
+  transform: translateY(4rem);
+}
+
+#home h2 {
+  font-size: 1.5rem;
+  padding-top: 6.3rem;
+  margin: auto;
+  width: 93%;
+  font-weight: 700;
+  -webkit-background-clip: text;
+  color: transparent;
+  background-image: url('../assets/h1Back.jpg');
+}
+
```

Comment:
- [ ] You `#home` section looks almost identical to the design guidelines. Bravo 👏 👏.  However the title heading of [ design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) is  a bit bolder. Please make sure to make your font look identical becuse according to [ project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#project-requirements) 

> You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:
>
> Colors.
>  - Typography: font face, size and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font
>  - similar to the one used in the original design).
>  - Layout: composition and space between elements.

Your design:

<img width="1510" alt="Screenshot 2023-03-17 at 9 38 29 PM" src="https://user-images.githubusercontent.com/44798044/225987551-2b1fccc4-45a9-44d1-819a-a96eeafb85fe.png">

Behance Design 

<img width="1034" alt="Screenshot 2023-03-17 at 11 11 45 PM" src="https://user-images.githubusercontent.com/44798044/225987642-e91767bd-9974-4a5d-af91-25421ce19e72.png">